packages/fluentui/react-northstar-fela-renderer/test/felaRenderer-test.tsx

```diff
@@ -1,20 +1,42 @@
 import { createFelaRenderer } from '@fluentui/react-northstar-fela-renderer';
 import { ICSSInJSStyle } from '@fluentui/styles';
-// @ts-ignore
-import { createSnapshot } from 'jest-react-fela';
+import { renderToString } from 'fela-tools';
```

Comment:
Wondering why nothing complains, but probably `fela-tools` should be added to `devDependencies` in `packages/fluentui/react-northstar-fela-renderer/package.json` or to a root `package.json`.

```diff
+"fela-tools": "^10.6.1",
```

Reply:
It's in react-northstar devDependencies, plus it's a dep of various other things, so maybe it's working implicitly for that reason.

Reply:
I added it to react-northstar-fela-renderer for now since we don't currently have any fela stuff at the root. But at some point those dev deps should probably all go to the root for consistency.